sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/state/RocksDBStateStoreProvider.scala

```diff
@@ -568,8 +577,8 @@ private[sql] class RocksDBStateStoreProvider
       if (!checkColFamilyExists(colFamilyName)) {
         val newColumnFamilyId = colFamilyId.incrementAndGet().toShort
```

Comment:
We can't generate the col family id on the executor. we need to do this on the driver